scenario-definitions.yaml

```diff
@@ -63,3 +67,11 @@ job_templates:
       DESKTOP: minimalx
       PUBLISH_HDD_1: 'opensuse-Tumbleweed-%ARCH%@%MACHINE%-%BUILD%.qcow2'
       PUBLISH_PFLASH_VARS: 'opensuse-Tumbleweed-%ARCH%@%MACHINE%-%BUILD%-uefi-vars.qcow2'
+  openqa_install_full+publish:
+    <<: *common
+    product: openqa-*-dev-full-x86_64
+    settings:
+      DESKTOP: minimalx
+      FULL: '1'
+      PUBLISH_HDD_1: 'opensuse-Tumbleweed-%ARCH%@%MACHINE%-%BUILD%-full.qcow2'
```

Comment:
Why do we need a different qcow name here? Previously it was using the same name